masonite/drivers/cache/CacheDiskDriver.py

```diff
@@ -168,6 +168,14 @@ def is_valid(self, key):
         return False
 
     def _create_directory(self, directory):
+        """creates a new dir.
+        
```

Comment:
```suggestion

```

Reply:
@Padmini-desai  please remove the whitespace in the line number, 72

Reply:
@vaibhavmule removed the whitespaces 
Please review the same

Reply:
sure, looking at it.